contrib/cirrus/logformatter

```diff
@@ -667,6 +667,11 @@ END_HTML
                 $subtest_name .= $2;
                 my $id = make_id($subtest_name, 'anchor');
                 $line = "<a name='t--$id'><h2 class=\"log-$subtest_status\">$line</h2></a>";
+
+                # Special case for tests that fail in [AfterEach] (and
+                # maybe BeforeEach too?). Do not include "TOP-LEVEL" in
+                # the actual failed-test anchor
+                $subtest_name = '' if $subtest_name =~ /^\s*TOP-LEVEL/;
```

Comment:
I think the TOP-LEVEL is inserted because we have global After/BeforeEach blocks since https://github.com/containers/podman/commit/ab29ff2f66216cfafa01add2c3238831a3824317

So basically if it is not inside a Describe() block it will say TOP-LEVEL if I understand the ginkgo structure correctly.
I.e. our SynchronizedBeforeSuite also shows up with TOP-LEVEL.

Reply:
Ah, that makes sense, thank you. That was worth a comment update and re-push, I've done so.